esphome/components/modbus_controller/modbus_controller.h

```diff
@@ -490,6 +493,8 @@ class ModbusController : public PollingComponent, public modbus::ModbusDevice {
   bool module_offline_;
   /// how many updates to skip if module is offline
   uint16_t offline_skip_updates_;
+  /// how many times we'll retry a command that got no response
+  uint8_t max_cmd_retries_;
```

Comment:
```suggestion
  uint8_t max_cmd_retries_{5};
```
If it's not set in the constructor, it should be initialised to something, even though in the usual case this will get overwritten later.

Reply:
I moved the initialization to the generated code, but I got your point, will fix. Is there any instrumentation that can "sync" both values? The one in C++ and the one in Python?